candy-machine/js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metaplex-foundation/mpl-candy-machine",
-  "version": "4.3.0",
+  "version": "4.4.0",
   "programVersion": "4.0.2",
```

Comment:
Should this be 4.1?

Reply:
No, the JS SDKs are versioned independently.